vue-project/src/components/Comment.vue

```diff
@@ -0,0 +1,51 @@
+<script>
+  export default {
+    data() {
+      return {
+        name: '',
+        email: '',
+        body: '',
+      }
+    },
+    emits: ['closeMenu', 'add-comment'],
+    methods: {
+      addComment() {
+        const newComment = {
+          'name': this.name,
+          'email': this.email,
+          'body': this.body,
+        };
+        this.$emit('add-comment', newComment);
+      }
+    }
+  }
+</script>
+<template>
+  <div class="block">
+    <form @submit.prevent="addComment">
+      <div class="field" data-cy="NameField">
+        <label class="label" for="comment-author-name-name">Author Name</label>
+        <div class="control has-icons-left has-icons-right">
+          <input type="text" name="name" id="comment-author-name-name"
+            placeholder="Name Surname" class="input" v-model="name" minlength="3" required>
```

Comment:
Fix code styles everywhere
```suggestion
          <input
            type="text"
            name="name"
            id="comment-author-name-name"
            placeholder="Name Surname"
            class="input"
            v-model="name"
            minlength="3"
            required
          >
```